apache/trafficcontrol
# Delivery Service Regional field: Optionally make Max Origin Connections per Cache Group

This PR resolves the Traffic Ops and Traffic Portal portions of #7097 by adding the `Regional` flag to Delivery Services for API versions 4 and up.
<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Ops

## What is the best way to verify this PR?

- Run the API tests
- Check the documentation

## PR submission checklist
